client/src/pages/AssignMates/AssignMatesPage.tsx

```diff
@@ -57,20 +68,110 @@ const move = <T,>(
   return result;
 };
 
+/**
+ * Create UnassignedMate from mate's syncs
+ * @param mate
+ * @returns UnassignedMate with lastSynced
+ */
+const populateLastSynced = (mate: Mate): UnassignedMate => {
```

Comment:
Having this since a mate's `syncs` can be empty in the future (even though it can't be right now)